mesh/test/vertex/TestHoneycombVertexMeshGenerator.hpp

```diff
@@ -39,6 +39,7 @@ OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <cxxtest/TestSuite.h>
 #include "HoneycombVertexMeshGenerator.hpp"
 #include "PetscSetupAndFinalize.hpp"
+#include "CheckpointArchiveTypes.hpp"
```

Comment:
also this one, as I don't think archiving is being done in added code.

Reply:
Resolved by commit 69c7ae2.